Dockerfile

```diff
@@ -315,19 +315,28 @@ ENTRYPOINT ["hack/dind"]
 FROM dev AS src
 COPY . /go/src/github.com/docker/docker
 
-FROM src AS build-binary
+FROM src AS binary-base
 ARG DOCKER_GITCOMMIT=HEAD
+ENV DOCKER_GITCOMMIT=${DOCKER_GITCOMMIT}
```

Comment:
I think autogen also needs the INIT_COMMIT 

oh; perhaps not; perhaps that was in my branch; could you double check?

Reply:
The autogen script handles this

```
source hack/dockerfile/install/runc.installer
source hack/dockerfile/install/tini.installer
source hack/dockerfile/install/containerd.installer
```